modules/metastore/src/NodeWrapper/Data.php

```diff
@@ -291,4 +291,14 @@ public function getOriginal() {
     }
   }
 
+  /**
+   * Save the "wrapped" node.
+   *
+   * Useful for some operations - usually recommended to use the metastore
+   * service's POST and PUT functions rather than saving the node directly.
+   */
+  public function save() {
+    $this->node->save();
+  }
+
```

Comment:
Is this being utilized anywhere in this PRs code? I'm not seeing it used but I could be wrong.